index.js

```diff
@@ -4,6 +4,8 @@ var _ = require('lodash');
 var Alexa = require('alexa-app');
 var app = new Alexa.app('carvis');
 var rideHelper = require('./ride-helper');
+var uberHelper = require('./uber-helper');
+var lyftHelper = require('./lyft-helper');
```

Comment:
why are these require statements here? doesnt look like we need them. can we take them out to avoid confusion? esp since i'll be refactoring this stuff. i guess i can take it out myself when i do that, lemme know if thats fine